049-SQLDedicatedPoolPerf/Coach/README.md

```diff
@@ -0,0 +1,51 @@
+# What The Hack - Dedicated SQL Pool performance best practices - Coach Guide
+
+## Introduction
+This "What the Hack" enables attendees to understand best practices developing, maintaining, and optimizing performance for their Synapse Dedicated SQL Pool. This hack simulates a real-world scenario where a cycle manufacturer company develops its new data warehouse leveraging Synapse Dedicated Sql pool. The goal is to apply all best practices to avoid performance bottlenecks during day by day activities. 
+
+During the hack, attendees will focus on:
+- Database Design – Best Practices
+- Query Design – Best practices and optimization
+- Performance Troubleshooting
+- Performance Monitoring
+
+## Student Resources
+Before the hack, it is the Coach's responsibility to download and package up the contents of the `/Student/Resources` folder of this hack into a "Resources.zip" file. The coach should then provide a copy of the Resources.zip file to all students at the start of the hack.
+
+**NOTE:** Students should not be given a link to the What The Hack repo before or during a hack. The student guide does NOT have any links to the Coach's guide or the What The Hack repo on GitHub.
+
+## Coach's Guides
+- Challenge 01: **[Ingest Data From Blob Storage](./Solution-01.md)**
+	 - In this challenge, attendees will import several parquet files from Blob Storage into their Dedicated SQL pool; 
+- Challenge 02: **[Queries Best Practice](./Solution-02.md)**
+	 - In this challenge, attendees will dig into best practices to consider writing a query for your data warehouse.
+- Challenge 03: **[Queries Behavior](./Solution-03.md)**
+	 - In this challenge, attendees will troubleshoot queries in stuck, not running queries and optimize query executions.
+- Challenge 04: **[Partitioned Table And Clustered Columnstore Indexes](./Solution-04.md)**
+	 - In this challenge, attendees will dig into the table partitioning strategy and its impact on performance. You will also get a deeper understanding of Columnstore Indexes behaviour
+- Challenge 05: **[Monitoring Workload](./Solution-05.md)**
+	 - In this challenge, attendees will understand how to monitor your workload to identify poor performing and failed queries
+
+## Coach Prerequisites 
+
```

Comment:
Please insert the following boiler-plate text which is going to be in the updated template:

"This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event.

These pre-reqs should be shared with a stakeholder in the organization that will be providing the Azure subscription(s) that will be used by the students."